VarLife.md

```diff
@@ -9,6 +9,12 @@ The [OTCA Metapixel](http://www.conwaylife.com/w/index.php?title=OTCA_metapixel)
 
 What [Life-like cellular automata](http://www.conwaylife.com/wiki/Life-like_cellular_automaton#Life-like_cellular_automata) means here is essentially that cells are born and cells survive according to how many of their eight neighbor cells are alive. The syntax for these rules is as follows: a B followed by the numbers of live neighbors that will cause a birth, then a slash, then an S followed by the numbers of live neighbors that will keep the cell alive. A bit wordy, so I think an example will help. The canonical Game of Life can be represented by the rule B23/S3, which says that any dead cell with two or three live neighbors will become alive and any live cell with three neighbors will remain alive. Otherwise, the cell dies.
 
+Despite being a 2048 x 2048 cell, the OTCA metapixel actually has a bounding box of 2058 x 2058 cells, the reason being that it overlaps by five cells in every direction with its neighbors. [[Why? What does this do? I have no idea.]] The birth and survival rules are encoded in a special section of cells at the left side of the metapixel, by the presence or absence of eaters in specific positions along two columns (one for birth, the other for survival). As for detecting the state of neighboring cells, here's how that happens:
```

Comment:
The overlap occurs because it is necessary for correct timing and neighbor metapixel updates. Without the overlap, the timing would be more complex, and would likely cause the metapixel to be larger.

Reply:
More specifically, the overlap is only on the diagonal neighbors. When a metapixel turns on, it will shoot out a glider to signal it's neighbors that it's on. However, if there is no neighbor, that glider must be eaten, or else that glider can fly off infinitely or break other metapixels. The overlap is necessary for the detection of the diagonal neighbors, but not for the orthogonal neighbors.